src/PageLayout/PageLayout.tsx

```diff
@@ -397,6 +397,7 @@ export type PageLayoutContentProps = {
   width?: keyof typeof contentWidths
   padding?: keyof typeof SPACING_MAP
   hidden?: boolean | ResponsiveValue<boolean>
+  as?: React.ElementType
```

Comment:
Sorry, I should have been more clear in my [previous comment](https://github.com/primer/react/pull/3154#discussion_r1167318494). I don't think the `PageLayout.Content` component should accept an `as` prop.

```suggestion
```